### ISSUE TITLE
Wrong output thrown after successive require

### ISSUE THREAD

**Issue by arichiardi** [ScalaConsultants/replumb#41]
Example:

```clojure
cljs.user=> (require '[cljs.pprint :refer [pprint]])
Could not parse ns form cljs.user - Referred var cljs.pprint/pprint does not exist
cljs.user=> (require 'clojure.string)
Could not parse ns form cljs.user - Referred var cljs.pprint/pprint does not exist
cljs.user=> 
```

Require still needs some adjustment.

**Comment by arichiardi** [comment]
Connected to #40.

**Comment by arichiardi** [comment]
Closed by #48.
